ui/src/ducks/app/volumes.js

```diff
@@ -234,6 +275,23 @@ export function* fetchPersistentVolumeClaims() {
     yield put(setPersistentVolumeClaimAction(result.body.items));
   }
 }
+export function* refreshPersistentVolumes() {
```

Comment:
LGTM but need unit test here.

Reply:
Thanks Carlito! I added unit test for refresh persistent volume.

Reply:
> LGTM but need unit test here.

Thanks Carlito! I added unit test for refresh persistent volume.